schema/draft/schema.json

```diff
@@ -144,6 +144,9 @@
                             },
                             {
                                 "$ref": "#/definitions/EmbeddedResource"
+                            },
+                            {
+                                "$ref": "#/definitions/ResourceReference"
```

Comment:
ResourceReference is missing `annotations`, which is especially useful for the `audience`. I know playwright wants to take helpful images/snapshots of the page for users to see but omit them from the model. Could we extend the ResourceReference interface instead here?